deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node-1GB-for-tests-only.yaml

```diff
@@ -158,14 +158,47 @@ spec:
               for (( i=1; i<=$SERVERS; i++ )); do
                   echo "server.$i=$NAME-$((i-1)).$DOMAIN:$SERVER_PORT:$ELECTION_PORT" >> /conf/zoo.cfg;
               done &&
+              if [[ $SERVERS -eq 1 ]]; then
```

Comment:
Do we need it for a single node?

Reply:
yes, cause on ACM side we always use `1-node.yaml` and just replace `replicas: X` and `SERVERS=X`